apps/settings/src/mixins/AppManagement.js

```diff
@@ -12,16 +12,76 @@ export default {
 			return this.app.groups.map(group => { return { id: group, name: group } })
 		},
 		installing() {
+			if (this.app?.app_api) {
+				return this.app && this.$store.getters['appApiApps/loading']('install')
+			}
 			return this.$store.getters.loading('install')
 		},
 		isLoading() {
+			if (this.app?.app_api) {
+				return this.app && this.$store.getters['appApiApps/loading'](this.app.id)
+			}
 			return this.app && this.$store.getters.loading(this.app.id)
 		},
+		isInitializing() {
+			if (this.app?.app_api) {
+				return this.app && Object.hasOwn(this.app?.status, 'action') && (this.app.status.action === 'init' || this.app.status.action === 'healthcheck')
+			}
+			return false
+		},
+		isDeploying() {
+			if (this.app?.app_api) {
+				return this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'deploy'
+			}
+			return false
+		},
+		isManualInstall() {
+			if (this.app?.app_api) {
+				return this.app?.daemon?.accepts_deploy_id === 'manual-install'
+			}
+			return false
+		},
+		updateButtonText() {
+			if (this.app?.daemon?.accepts_deploy_id === 'manual-install') {
+				return t('app_api', 'manual-install apps cannot be updated')
+			}
+			return ''
+		},
 		enableButtonText() {
-			if (this.app.needsDownload) {
-				return t('settings', 'Download and enable')
+			if (this.app?.app_api) {
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'deploy') {
+					return t('app_api', '{progress}% Deploying', { progress: this.app.status?.deploy })
+				}
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'init') {
+					return t('app_api', '{progress}% Initializing', { progress: this.app.status?.init })
+				}
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'healthcheck') {
+					return t('app_api', 'Healthchecking')
+				}
+				if (this.app.needsDownload) {
+					return t('app_api', 'Deploy and Enable')
+				}
+				return t('app_api', 'Enable')
+			} else {
+				if (this.app.needsDownload) {
+					return t('settings', 'Download and enable')
+				}
+				return t('settings', 'Enable')
+			}
+		},
+		disableButtonText() {
+			if (this.app?.app_api) {
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'deploy') {
+					return t('app_api', '{progress}% Deploying', { progress: this.app.status?.deploy })
+				}
+				if (this.app && Object.hasOwn(this.app?.status, 'action') && this.app.status.action === 'init') {
+					return t('app_api', '{progress}% Initializing', { progress: this.app.status?.init })
```

Comment:
same with the `hasOwn` as above, but also the string has wrong case:

```suggestion
					return t('app_api', '{progress}% initializing …', { progress: this.app.status?.init })
```